scripts/per-prompt-context-injection.py

```diff
@@ -10,4 +10,11 @@
 print("📌 fake-driven-testing: If not loaded, load now. Always abide by its rules.")
 print("🚫 No direct Bash for: pytest/pyright/ruff/prettier/make/gt")
 print("✅ Use Task(subagent_type='devrun') instead.")
-print("📌 dignified-python: If not loaded, load now. Always abide by its rules.")
+print(
+    """📌 dignified-python: CRITICAL RULES (examples - full skill has more):
+❌ NO try/except for control flow (use LBYL - check conditions first)
+❌ NO default parameter values (no `foo: bool = False`)
+❌ NO mutable/non-frozen dataclasses (always `@dataclass(frozen=True)`)
+⚠️ MANDATORY: Load and READ the full dignified-python skill documents.
+   These are examples only. You MUST strictly abide by ALL rules in the skill."""
```

Comment:
The above  are simplified examples only. Full text is in the skill.

Reply:
Resolved via /erk:pr-address at 2025-12-30 05:29 AM PST

_Addressed via `/erk:pr-address` at 2025-12-30 05:29 PST_